### PR TITLE
feat:Add an information in task history when adding images - MEED-2437 - Meeds-io/MIPs#53

### PR DESCRIPTION
**Pull request by SaraBoutej**
This PR adds an new log to the task changes drawer when attaching a new image to the task description.

### CODE REVIEW

File: services/src/main/java/org/exoplatform/task/integration/TaskAttachmentLoggingListener.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package org.exoplatform.task.integration;
```
Comment (boubaker) [inline services/src/main/java/org/exoplatform/task/integration/TaskAttachmentLoggingListener.java:19]: Missing license Header

File: services/src/main/java/org/exoplatform/task/integration/TaskAttachmentLoggingListener.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    ObjectAttachmentId objectAttachment = event.getData();
+
+    if (objectAttachment == null) {
+      return;
+    }
+    if (!objectAttachment.getObjectType().equals(ATTACHMENT_TASK_OBJECT_TYPE)) {
+      return;
+    }
+    taskService.addTaskLog(Long.parseLong(objectAttachment.getObjectId()), username, "attach_image", "");
+  }
```
Comment (boubaker) [inline services/src/main/java/org/exoplatform/task/integration/TaskAttachmentLoggingListener.java]:
```suggestion
    if (objectAttachment != null && ATTACHMENT_TASK_OBJECT_TYPE.equals(objectAttachment.getObjectType())) {
      taskService.addTaskLog(Long.parseLong(objectAttachment.getObjectId()), username, "attach_image", "");
    }
  }
```

File: services/src/main/java/org/exoplatform/task/integration/TaskAttachmentLoggingListener.java (hunk 1 above)
Comment (SaraBoutej) [reply services/src/main/java/org/exoplatform/task/integration/TaskAttachmentLoggingListener.java:19]: added!

File: services/src/main/java/org/exoplatform/task/integration/TaskAttachmentLoggingListener.java (hunk 2 above)
Comment (SaraBoutej) [reply services/src/main/java/org/exoplatform/task/integration/TaskAttachmentLoggingListener.java]: done

**Review by boubaker — APPROVED** [review]

**Pull request merged**